packages/modal-ui-js/README.md

```diff
@@ -0,0 +1,11 @@
+# modal-ui-js
```

Comment:
We will need to update the Readme file if it's the same as the old modal just copy it here too.